tests/test_import_migration.py

```diff
@@ -14,7 +14,13 @@ def test_simple_from_import():
 def test_from_to_from_import():
     red = RedBaron("from flask.ext.foo import bar")
     output = migrate.fix_tester(red)
-    assert output == "from flask_foo import bar as bar"
+    assert output == "from flask_foo import bar"
```

Comment:
It may be easier to understand if these test changes (here and L20) go in 99f9ff3.